rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -28,12 +30,28 @@ include::rpa-home::partial$windows-versions-support.adoc[]
 
 |===
 
-=== Requirements for Secure Session Technology
+=== System Settings
+
+* Display resolution: 
+** Minimum: 1280 x 1080
+** Recommended: Full HD 1080p (1920 × 1080) or higher
+* Scaling of text and other items on the screen: 100 %
+* _Clear Type_: turned off
+
+=== Requirements for the Secure Session Technology
+
+Secure Sessions are hidden Desktop sessions. They run in the background and only the bot can access them. An additional advantage of using Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
+
+To use Secure Sessions you must ensure that all prerequisites are met and all responsible persons are informed.
 
-* Windows logins used for secure sessions have privileges to open a local RDP session on `127.0.0.2`.
-* 64-bit Operating System.
-* Dedicated Windows Server machine with enough resources to execute several processes simultaneously (2 GB or more RAM per session)
-* The RPA Bot service runs under the _Local System account_ (by default, the setup configures RPA Bot to use this account).
+* The RPA Bot service runs under the _Local System_ account per default.
+** If you want to use another account, ensure that it has permission to run Windows services.
+* The RPA Bot service runs processes under user accounts existing on the same computer or within the same domain.
+** We recommend using dedicated accounts and a user naming scheme to differentiate between local users and domain users and between human users and bot users.
+** User accounts used by the bot to run processes must belong to the _Remote Desktop Users_ group and must be allowed to establish a local RDP session via the local loopback address 127.0.0.2.
+* If you want a bot to run more than one (Windows 10, Windows 11) or two (Windows Server) Secure Sessions simultaneously, make sure that a terminal server software with enough licensed users is installed on the computer where the bot is installed. The computer must have enough resources to execute several processes simultaneously: 2 GB or more RAM per session. 
```

Comment:
```suggestion
* To enable a bot to run more than one (Windows 10, Windows 11) or two (Windows Server) Secure Sessions simultaneously, make sure that a terminal server software with enough licensed users is installed on the computer where the bot is installed. The computer must have enough resources to execute several processes simultaneously: 2 GB or more RAM per session. 
```

Reply:
What does enough (licensed users) mean in this case? How do they determine that?

Reply:
The computer must have at least 2 GB of RAM to run each secure session?

Are processes = sessions? Or secure sessions = several processes?

Reply:
The Terminal Server (simulation) software enables the operting system to run more than one (or two) sessions in parallel.
A session is what you get when you log in to the computer.  You have a Desktop that you can see and the system to run your application and you identify by your credentials. If you are logged in to your PC, no one else can log in at the same time.
The Terminal Server software takes away this restriction. But still Microsoft has licensed your OS to be used only by one person at the same time. 
So to stay within legal realms you have to buy an extra license for the number of users that should be able to log in in parallel.
The bot uses the credentials of a user on the computer. (He can log in with the same user multiple times in parallel or use multiple users in parallel)
Only one process can be run in one session. But one process can be run in multiple sessions.
Each session will use at least 2 GB RAM.

Reply:
I assume that if you buy a terminal server software they tell you to acquire user licenses for the number of users that should be able to log in in parallel. 
We do not maintain or (anymore) ship that software and should not advertise one.

Reply:
I agree that this is a bit vague, but I don't know how to precise that further within our limited universe.